docs/source/usage_guides/fsdp.md

```diff
@@ -96,7 +96,7 @@ all-gather while executing in the forward pass. only use with Static graphs.
 Useful in cases such as parameter-efficient fine-tuning. 
 Please refer this [blog](https://dev-discuss.pytorch.org/t/rethinking-pytorch-fully-sharded-data-parallel-fsdp-from-first-principles/1019)
 
-`CPU RAM Efficient Model loading`: If True, only the first process loads the pretrained model checkoint while all other processes have empty weights. Only applicable for 🤗 Transformers models. When using this, `Sync Module States` needs to True. 
+`CPU RAM Efficient Model loading`: If True, only the first process loads the pretrained model checkoint while all other processes have empty weights. Only applicable for 🤗 Transformers models. When using this, `Sync Module States` needs to True else all the processes expect the main process would have random empty weights leading to unexpected behaviour during training.
```

Comment:
"needs to True" => "needs to **be** True"

I think there was a misunderstanding here, maybe I didn't explain myself well. What I meant earlier with adding an explanation was not related to why `fsdp_sync_module_states` must be `True` if `fsdp_cpu_ram_efficient_loading` is set to `True`. What I meant is adding an explanation under what circumstances users should set `fsdp_cpu_ram_efficient_loading=False`.

Reply:
Done